src/containers/ReportArticle/ReportArticle.test.js

```diff
@@ -0,0 +1,65 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { ReportArticle } from './index';
+import { fetchTypesResponse, user } from './state/helper';
+import { article } from '../../utils/testHelpers';
+
+const mockStore = configureStore();
```

Comment:
@SrMoffat the `testHelpers` script defines a `mockStore` helper. You could make use of that to prevent repetition of tasks.

Reply:
I have realised I was not even using the store in the tests other than on the `beforeEach` function with `store.clearAction` so I have gotten rid of it as a whole.